lib/scheduler.js

```diff
@@ -32,7 +32,7 @@ class Scheduler {
 
   run() {
     const offlinePlugin = this.serverless.pluginManager.getPlugins()
-      .find((p) => p.constructor && p.constructor.name === "Offline");
+      .find((p) => p.constructor && p.constructor.name === "ServerlessOffline");
```

Comment:
Please update to support both versions of the constructor name

Reply:
ok done...